README.md

```diff
@@ -26,3 +26,9 @@ example-formula
 ## Packaging Information
 
 For information about packaging Formulas as RPM packages (including a `spec` file template) please refer to the [packaging guide](https://github.com/SUSE/salt-formulas/wiki/Packaging-Guide).
+
+## Stable branches
```

Comment:
```suggestion
## Stable Branches
```
Maybe for consistency with the other headings in this README we might want to uppercase this one, too?